components/public-api-server/pkg/apiv1/tokens_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package apiv1
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	connect "github.com/bufbuild/connect-go"
+	"github.com/gitpod-io/gitpod/common-go/experiments"
+	v1 "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1"
+	"github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1/v1connect"
+	protocol "github.com/gitpod-io/gitpod/gitpod-protocol"
+	"github.com/gitpod-io/gitpod/public-api-server/pkg/auth"
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTokensService_CreatePersonalAccessTokenWithoutFeatureFlag(t *testing.T) {
+	t.Run("returns a personal access token", func(t *testing.T) {
+		serverMock, client := setupTokensService(t)
+
+		user := &protocol.User{
+			ID:           uuid.New().String(),
+			Name:         "Someone",
+			CreationDate: "2022-11-15T10:10:10.000Z",
+		}
+
+		serverMock.EXPECT().GetLoggedInUser(gomock.Any()).Return(user, nil)
+
+		_, err := client.CreatePersonalAccessToken(context.Background(), &connect.Request[v1.CreatePersonalAccessTokenRequest]{})
+		require.Error(t, err, "This feature is currently in beta. If you would like to be part of the beta, please contact us.")
```

Comment:
I would also assert the status code here

```suggestion
		require.Equal(t, connect.PermissionDenied, connect.CodeOf(err))
```